### PR TITLE
Transaction details minor improvements.

### PR DESCRIPTION
**Pull request by kukabi**
Transaction details should display the transaction from the list if it's opened from the list, or load the transaction fresh if it's opened through a push notification. Transaction details small layout fix. Bump build number.

### CODE REVIEW

File: app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt [hunk 1]
```diff
@@ -135,18 +143,30 @@ internal class TxDetailsActivity : AppCompatActivity(), ServiceConnection {
...
         setupUI()
-        tracker.screen(path = "/home/tx_details", title = "Transaction Details")
-    }
 
-    override fun onStart() {
-        super.onStart()
-        // start service if not started yet
+        val savedTx = savedInstanceState?.getParcelable<Tx>(TX_EXTRA_KEY)
+        val intentTx = intent.getParcelableExtra<Tx>(TX_EXTRA_KEY)
+        when {
```
Comment (nyarian) [inline app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt:149]: could it be `if (savedTx != null || intentTx != null)` with subsequent `tx = savedTx ?: intentTx`?

File: app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt [hunk 2]
```diff
@@ -135,18 +143,30 @@ internal class TxDetailsActivity : AppCompatActivity(), ServiceConnection {
...
+                enableCTAs()
+            }
+        }
         if (walletService == null) {
             // bind to service
             val bindIntent = Intent(this, WalletService::class.java)
             Logger.d("Issuing bindService (${System.currentTimeMillis()})")
             bindService(bindIntent, this, Context.BIND_AUTO_CREATE)
         }
+        tracker.screen(path = "/home/tx_details", title = "Transaction Details")
```
Comment (nyarian) [inline app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt:169]: won't it be called on every config change?

Comment (kukabi) [reply app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt:169]: correct, fixing.

File: app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt (hunk 1 above)
Comment (kukabi) [reply app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt:149]: how about
```kotlin
(savedInstanceState?.getParcelable(TX_EXTRA_KEY)
    ?: intent.getParcelableExtra<Tx>(TX_EXTRA_KEY))?.let {
    tx = it
    bindTxData()
    observeTxUpdates()
    enableCTAs()
}
```

Comment (kukabi) [reply app/src/main/java/com/tari/android/wallet/ui/activity/tx/TxDetailsActivity.kt:149]: or
```kotlin
val savedTx = savedInstanceState?.getParcelable<Tx>(TX_EXTRA_KEY)
val intentTx =  intent.getParcelableExtra<Tx>(TX_EXTRA_KEY)
(savedTx ?: intentTx)?.let {
    tx = it
    bindTxData()
    observeTxUpdates()
    enableCTAs()
}
```

**Pull request merged**